Dockerfile

```diff
@@ -48,7 +48,8 @@ RUN curl -sfL https://github.com/duplicati/duplicati/releases/download/v2.0.5.11
 RUN apt -y install /tmp/duplicati.deb
 
 # Requirements for UI tests
-RUN DEBIAN_FRONTEND="noninteractive" apt-get -y install chromium xorg xvfb gtk2-engines-pixbuf dbus-x11 xfonts-base xfonts-100dpi xfonts-75dpi xfonts-cyrillic xfonts-scalable imagemagick x11-apps
+RUN DEBIAN_FRONTEND="noninteractive" apt install -y chromium xorg xvfb gtk2-engines-pixbuf dbus-x11 xfonts-base xfonts-100dpi xfonts-75dpi xfonts-cyrillic xfonts-scalable imagemagick x11-apps
+RUN apt install -y brotli python3-pip && pip3 install selenium
```

Comment:
Why do we need selenium python package?

Reply:
I created some UI tests with selenium and plan more. Selenium is build for testing and more feature rich than gorod. After I got everything running I am able to remove chromium xorg and so on in line 51.

Reply:
As I understand, this introduces a new tool and things that people need to install to develop? Does the tooling work across all OS-s? We're already having trouble with simple things like bash & docker tests not working properly on Windows.

Reply:
It is not a tool that every developer needs to develop. Yes it works with all OS-s including Freebsd and solaris. There are drivers for all common browsers Chrome, Edge, Safari, Firefox and so on. I should create tests whether the Webui shows what it should, for this I need a framework that can do this.

Reply:
pip3 and selenium are removed, but I still need brotli.